R/PACo.r

```diff
@@ -23,9 +23,12 @@ PACo <- function(D, nperm=1000, seed=NA, method="r0")
    m2ss <- proc$ss
    pvalue <- 0
    if(!is.na(seed)) set.seed(seed)
+   # Create randomised matrices
+   null_model <- vegan::nullmodel (D$HP, method)
+   randomised_matrices <- simulate (null_model, nsim = nperm)
```

Comment:
simulate is in the stats package 

- Fer

On Sat, Feb 21, 2015 at 3:08 AM, Timothée Poisot <notifications@github.com>
wrote:

>> @@ -23,9 +23,12 @@ PACo <- function(D, nperm=1000, seed=NA, method="r0")
>>     m2ss <- proc$ss
>>     pvalue <- 0
>>     if(!is.na(seed)) set.seed(seed)
>> +   # Create randomised matrices
>> +   null_model <- vegan::nullmodel (D$HP, method)
>> +   randomised_matrices <- simulate (null_model, nsim = nperm)
> Where does `simulate` come from? Should it be `vegan::simulate` ?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/PoisotLab/paco/pull/7/files#r25070937